packages/website/components/navbar.js

```diff
@@ -1,17 +1,16 @@
-import { useCallback, useMemo, useRef, useState } from 'react'
-import { useRouter } from 'next/router'
-import Link from 'next/link'
-import { getMagic } from '../lib/magic.js'
-import countly from '../lib/countly'
-import { useQueryClient } from 'react-query'
-import Button from './button.js'
-import { useResizeObserver } from '../hooks/resize-observer'
-import clsx from 'clsx'
-import Hamburger from '../icons/hamburger'
+import { useCallback, useMemo, useRef, useState } from "react";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import { getMagic } from "../lib/magic.js";
+import countly from "../lib/countly";
+import { useQueryClient } from "react-query";
+import Button from "./button.js";
+import clsx from "clsx";
+import Hamburger from "../icons/hamburger";
 
-import Logo from '../icons/w3storage-logo'
-import Cross from '../icons/cross'
-import Loading from './loading.js'
+import Logo from "../icons/w3storage-logo";
```

Comment:
looks like you have prettier changing single quotes to double quotes and adding semicolons. Let's not do that in this pr if possible 🙏

Reply:
This issue should be resolved now!